OpenEdXMobile/src/main/java/org/edx/mobile/view/RegisterActivity.java

```diff
@@ -285,13 +285,17 @@ public void onException(Exception ex) {
                             if (key.equalsIgnoreCase(fieldView.getField().getName())) {
                                 List<RegisterResponseFieldError> error = messageBody.get(key);
                                 showErrorOnField(error, fieldView);
-                                fieldErrorShown = true;
+                                if (!fieldErrorShown) {
```

Comment:
Its upto you to change or not, but I'd still say we have a more descriptive and scope-specific name.